snap/snapcraft.yaml

```diff
@@ -63,6 +63,13 @@ description: |
 
   The DCGM snap does not currently include the performance testing tool `dcgmproftester` in order to limit the size of the snap.
 
+  **Cryptography**
```

Comment:
I think it's worth mentioning how the CUDA keyring is used (i.e. to verify the signature of the DCGM deb package)